.travis.yml

```diff
@@ -44,7 +44,7 @@ env:
   - SPREAD_PATH=/tmp/spread
   matrix:
   - SYSTEM=google:ubuntu-devel-64 VARIANT=amd64
-  - SYSTEM=google:fedora-rawhide-64 VARIANT=amd64
+#  - SYSTEM=google:fedora-rawhide-64 VARIANT=amd64
```

Comment:
How about allowing it to fail instead? https://github.com/MirServer/mir/commit/77c4ef82a7c997d76dd1259b4d2c1c0fc8f54e58

Reply:
How's this?